.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -22,7 +22,7 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Environment (please complete the following information):**
+**Enviornment (please fill out the following information)**
```

Comment:
Please revert this, it introduces a typo.

Reply:
Fixed the typo